api/v1beta2/imageupdateautomation_types.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2020 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta2
```

Comment:
I see no API changes here, other than adding another value for the enum which doesn't really count. The API version bump is not justified.

Reply:
OK, then good news... Wasn't sure of this. I will revert.